src/index.d.ts

```diff
@@ -1000,6 +1020,16 @@ declare namespace admin.auth {
      * to the tenant corresponding to that `TenantAwareAuth` instance's tenant ID.
      */
     tenantId?: string | null;
+
+    multiFactor?: {
+      enrolledFactors: Array<{
```

Comment:
Does it make sense to reduce this duplication by introducing an interface? Like the `SecondFactor` type used in the implementation?

Reply:
Yeah I think we can define an interface for this. I can do it in the next PR. This is going to be modified and more documentation added later anyway.